dashboard/utils.py

```diff
@@ -155,3 +170,119 @@ def update_header_diffs(scan):
 
     scan.update_header_diffs(ignore=ignore, tolerance=tolerance,
                              bvals=check_bvals)
+
+
+def delete_session(session):
+    config = datman.config.config(study=session.get_study().id)
+
+    files = [scan.name for scan in session.scans]
+    for path_key in ['dcm', 'nii', 'mnc', 'nrrd', 'jsons']:
+        delete(config, path_key, folder=str(session.timepoint), files=files)
+
+    delete(config, 'dicom', files=['{}.zip'.format(str(session))])
+    delete(config, 'resources', folder=str(session))
+
+    timepoint = session.timepoint
+    if not timepoint.bids_name:
+        return
+
+    for scan in session.scans:
+        delete_bids(config, timepoint.bids_name, timepoint.bids_session, scan)
+
+
+def delete_scan(scan):
+    config = datman.config.config(study=scan.get_study().id)
+
+    for path_key in ['dcm', 'nii', 'mnc', 'nrrd', 'jsons']:
```

Comment:
Should we also check the `std` folder?

Reply:
Ahh good point! I forgot about gold standards... Thanks for catching that! I'll also try to make sure I include screenshots more consistently when I change the UI haha

Reply:
Ok, I added gold standard folder clean up but the records in the database wont be deleted. I deliberately left the gold standard records completely separate from the scans records so that we could keep a history of changes to the gold standards, and the most recent ones will only ever be used anyway.

Reply:
Good call! I didn't consider that.